app/javascript/controllers/simple_tree_controller.js

```diff
@@ -33,6 +23,29 @@ export default class extends Controller {
     event.target.nextElementSibling.nextElementSibling.classList.toggle('hidden')
   }
 
+  #centerTreeView() {
+    setTimeout(() => {
+      const location = window.location.href;
+      const isTreeViewPage = ['classes', 'properties', 'schemes', 'collections', 'instances']
```

Comment:
here you have two options: 

1. replace this with a stimulus value, so that it is still generic.
2. enable that behavior fo centering only if the tree is displayed and not hidden, I found this that can helpful on that https://stimulus-use.github.io/stimulus-use/#/use-visibility,  and have something like:
```export default class extends Controller {
  connect() {
    useVisibility(this)
  }

  visible() {
    this.#centerTreeView()
  }

  invisible() {
    // do nothing
  }
}
```

Reply:
Done using `useVisibility`

Reply:
Did it work?